src/components/Button/Button.module.scss

```diff
@@ -1,3 +1,4 @@
+@import "~monday-ui-style/dist/mixins";
```

Comment:
When the size of the text depend on the size of the component I choosed to use mixins because I felt the result is cleaner